src/NuGet.Services.BasicSearch.Cloud/NuGet.Services.BasicSearch.Cloud.ccproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>2.7</ProductVersion>
+    <ProductVersion>2.9</ProductVersion>
```

Comment:
I don't mind, I moved to 2.9 for my local build, you can roll it back before publishing. Moving to 2.9 is lower pri at this point